code/modules/mob/living/simple_animal/parrot.dm

```diff
@@ -685,13 +685,12 @@
 	clean_speak = list(
 		"Poly wanna cracker!",
 		"Check the crystal, you chucklefucks!",
-		"Check the tesla, you shits!",
 		"STOP HOT-WIRING THE ENGINE, FUCKING CHRIST!",
 		"Wire the solars, you lazy bums!",
 		"WHO TOOK THE DAMN HARDSUITS?",
 		"OH GOD ITS ABOUT TO DELAMINATE CALL THE SHUTTLE",
 		"Why are there so many atmos alerts?",
-		"OH GOD WHY WOULD YOU PUT CARBON-DIOXIDE IN THE SM?",
+		"OH GOD WHY WOULD YOU PUT PLASMA IN THE SM?",
```

Comment:
```suggestion
		"OH GOD WHY WOULD YOU PUT PLASMA IN THE SM?",
		"Why is nobody watching the engine?",
		"Maybe the SM would produce more power if we fed it some clowns.",
		"Everyone else dusted when they touched the SM, but I am sure you will be different.",
		"I asked the mime if they turned off the scrubbers, but they didn't say a word.",
		"This engine setup meets all safety requirements.",
		"The natural diet of the SM consists of Chief Engineers.",
```

Reply:
"don't eat the forbidden Nacho for fucks sake"

Reply:
Wasn't planning on adding any more, but since you're got them all here...